pandas/tests/indexes/datetimes/test_partial_slicing.py

```diff
@@ -331,6 +331,38 @@ def test_partial_slice_doesnt_require_monotonicity(self):
         with pytest.raises(KeyError, match=r"Timestamp\('2014-01-10 00:00:00'\)"):
             nonmonotonic.loc[timestamp:]
 
+    @pytest.mark.parametrize("indexer_end", [None, "2020-01-02 23:59:59.999999999"])
+    def test_loc_getitem_partial_slice_non_monotonicity(self, indexer_end):
+        # GH#33146
+        df = DataFrame(
```

Comment:
if the test involves a DataFrame, it doesnt belong in tests.indexes (i know everything in this file is a counterexample, but im in the process of moving these tests to the correct locations) 

if you want to write a test that directly tets DTI.slice_locs, that'd go in tests.index.datetimes.test_indexing.TestSliceLocs

Reply:
Where should I place it after parametrizing over frame_or_series?

Reply:
tests.indexing.test_loc, probably in TestPartialStringSlicing

Reply:
Moved the test